statistics_msgs/msg/StatisticDataType.msg

```diff
@@ -0,0 +1,18 @@
+#############################################
+# This file contains the commonly used constants for the statistics data type.
+#
+# The value 0 is reserved for unitialized statistic message data type.
+# Range of values: [0, 255].
+# Unallowed values: any value that is not specified in this file.
+#
+#############################################
+
+# Constant for uninitialized
+uint8 STATISTICS_DATA_TYPE_UNINITIALIZED = 0
+
+# Allowed values
```

Comment:
It would be good to check the generated code on how these comments appear in docblocks. I would guess this one will be associated with the first enum which seems weird.